### DISCUSSION TITLE
use with vim/plug

### DISCUSSION THREAD

**Discussion by atefBB** [NeogitOrg/neogit#1413] [Q&A]
Hi, how to install this plugin with vim/plug ?

**Comment by CKolkey** [comment]
Probably something like:
```vim
call plug#begin()

" List your plugins here
Plug 'neogitorg/neogit'

call plug#end()

lua "require('neogit').setup()"
```

Not sure, though. I haven't used Plug in years.

**Reply by atefBB** [reply]
I'll test it and return to u with my feedback ! thx !

**Reply by ByteCommander7** [reply]
> I'll test it and return to u with my feedback ! thx !

Did his solution work?

**Reply by atefBB** [reply]
> > I'll test it and return to u with my feedback ! thx !
> 
> Did his solution work?

Sorry I've abandoned using this plugin !
